debian/copyright

```diff
@@ -0,0 +1,37 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: restic
+Source: https://github.com/restic/restic
+
+Files: *
+Copyright: 2020 iXsystems Inc
```

Comment:
Please remove references to BSD licenses. We're not altering this package in any way so we'll just  reference upstream copyright/license.

Reply:
Is there an example of how this file should look like?

Reply:
@themylogin Just leave the `Format:, Upstream-Name:` and `Source:` lines and remove everything else?